test/battle/move_effect/chilly_reception.c

```diff
@@ -0,0 +1,79 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_CHILLY_RECEPTION].effect == EFFECT_CHILLY_RECEPTION);
+}
+
+SINGLE_BATTLE_TEST("Chilly Reception switches the user out")
+{
+    GIVEN {
+        PLAYER(SPECIES_SLOWKING_GALARIAN);
+        PLAYER(SPECIES_SLOWPOKE_GALARIAN);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_CHILLY_RECEPTION); SEND_OUT(player, 1); }
+    } SCENE {
+        MESSAGE("Slowking is preparing to tell a chillingly bad joke!");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CHILLY_RECEPTION, player);
+        MESSAGE("It started to snow!");
+        MESSAGE("Go! Slowpoke!");
+        ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_SNOW_CONTINUES);
+    }
+}
+
+SINGLE_BATTLE_TEST("Chilly Reception switches the user out, even if the weather does not change")
+{
+    GIVEN {
+        PLAYER(SPECIES_SLOWKING_GALARIAN);
+        PLAYER(SPECIES_SLOWPOKE_GALARIAN);
+        OPPONENT(SPECIES_KYOGRE_PRIMAL);
+    } WHEN {
+        TURN { MOVE(player, MOVE_CHILLY_RECEPTION); SEND_OUT(player, 1); }
+    } SCENE {
+        MESSAGE("Slowking is preparing to tell a chillingly bad joke!");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CHILLY_RECEPTION, player);
+        MESSAGE("There is no relief from this heavy rain!");
+        MESSAGE("Go! Slowpoke!");
+    }
+}
+
+SINGLE_BATTLE_TEST("Chilly Reception does not switch the user out if no replacements")
+{
+    GIVEN {
+        PLAYER(SPECIES_SLOWKING_GALARIAN);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_CHILLY_RECEPTION); }
+    } SCENE {
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CHILLY_RECEPTION, player);
+    }
+}
+
+SINGLE_BATTLE_TEST("Chilly Reception does not switch the user out if replacements fainted")
+{
+    GIVEN {
+        PLAYER(SPECIES_SLOWKING_GALARIAN);
+        PLAYER(SPECIES_SLOWPOKE_GALARIAN) { HP(0); }
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_CHILLY_RECEPTION); }
+    } SCENE {
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CHILLY_RECEPTION, player);
```

Comment:
Please add the `{pokemon} went back to {player}` message but negate it with `NOT` before the message. 

If possible add it everywhere where a switch is not happening.